core/script.runtime/src/kotlin/script/templates/standard/templates.kt

```diff
@@ -16,18 +16,25 @@
 
 package kotlin.script.templates.standard
 
+// discuss
+//
+// These are some 'basic' script templates
+// Should we keep them here?
```

Comment:
Of course it is, but it isn't an argument as to whether we should keep it :) The string array is a trivial concept and we require everyone starting Kotlin programming to understand that the entrypoint is always `fun main(args: Array<String>)` anyway. We didn't introduce `fun main()` just because it's easier, there was no demand. So I hope that almost everyone will immediately understand why scripts need arguments, and if a script doesn't need arguments, it can just ignore them.